apps/mull-ui/src/app/components/pill-options/pill-options.tsx

```diff
@@ -17,16 +17,18 @@ export const PillOptions = ({ options, onChange, active }: PillOptionsProps) =>
   return (
     <div className="pill-options">
       {options.map((option, idx) => (
-        <div
+        <button
```

Comment:
Comment for 86.4: I know this is the fix for #309, however it still submits the form once I click on any of the pillbox options.

Reply:
and you're right, good catch